spec/unit/message_spec.rb

```diff
@@ -14,21 +14,48 @@
     expect(msg._data).to eq data
   end
 
-  it 'double to_json\'s the identifier and the data' do
-    command = 'hi',
-              identifier = { "hi": 'there' },
-              data = { "data": 'boo' }
-
-    expected = {
-      command: command,
-      identifier: identifier.to_json,
-      data: data.to_json
-    }
-
-    msg = ActionCableClient::Message.new(command, identifier, data)
-
-    expected = expected.to_json
-
-    expect(msg.to_json).to eq expected
+  describe '#to_json' do
+    context 'when data is present' do
+      non_blank_data = [Object.new, true, 0, 1, 'a', [nil], { nil => 0 },
+                        Time.now]
+
+      non_blank_data.each do |data|
+        it "double the identifier and the data if the value would be #{data}" do
```

Comment:
should inspect be used here as well?

Reply:
Yes. It produces almost the same output, just `a` vs `"a"` but should improve readability, I guess.